mobile_back_qti/make/qti_backend.mk

```diff
@@ -22,7 +22,7 @@ ifeq (${WITH_QTI},1)
   local_snpe_sdk_root=$(shell echo mobile_back_qti/snpe-* | awk '{print $$NF}')
   $(info detected SNPE SDK: ${local_snpe_sdk_root})
   backend_qti_android_files=${BAZEL_LINKS_PREFIX}bin/mobile_back_qti/cpp/backend_qti/libqtibackend.so \
-    ${BAZEL_LINKS_PREFIX}bin/flutter/android/commonlibs/lib_arm64/libc++_shared.so \
+    ${local_snpe_sdk_root}/lib/aarch64-android-clang8.0/libc++_shared.so \
```

Comment:
Why do we need this change?

This change is incompatible with Samsung backend. 

Samsung backend also requires `libc++_shared.so` file.
If the file provided by SNPE SDK is different from `flutter/android/commonlibs/lib_arm64/libc++_shared.so` it may be incompatible with Samsung backend by itself.
And even if it's compatible there will be troubles building the app because currently we copy all files listed by all backends, and the copy command will complain about 2 files with identical names.

Reply:
somehow flutter/android/commonlibs path was not working with docker flutter steps. 
I will try to see how to map bazel-bin from inside docker.

Reply:
found that flutter/commonlibs was not built when building only qti backend. 
So, moved bulding of flutter commonlibs to tflitebackend, which is enabled by default.

Reply:
Could you move commonlibs bazel target to both `backend_qti_android_target` and `backend_samsung_android_target` configs, please?
Bazel doesn't mind duplicated targets.
And it's technically possible to build the app without tflite backend.

Reply:
done.